### DISCUSSION TITLE
BusinessReport csv download using API

### DISCUSSION THREAD

**Discussion by CoolDev0723** [amzn/ads-advanced-tools-docs#199] [Q&A]
Business Reports |Detail Page Sales and Traffic By Parent Item

https://sellercentral-japan.amazon.com/business-reports/ref=xx_sitemetric_dnav_xx#/report?id=102%3ADetailSalesTrafficByParentItem&chartCols=&columns=0%2F1%2F2%2F7%2F8%2F13%2F14%2F19%2F20%2F25%2F26%2F27%2F28%2F29%2F30%2F31%2F32%2F33%2F34%2F35%2F36&fromDate=2023-11-01&toDate=2023-11-23

Can I download this csv using API?
I checked the API documentation, but couldn't find a matching endpoint.
I would appreciate it if anyone could teach me.
Thanks in advance.

![image](https://github.com/amzn/ads-advanced-tools-docs/assets/54260086/7c43b3a8-938f-4f71-b9aa-f38a95911a85)

**Comment by drewxa** [comment]
Certainly, you can access this data using the Seller Partner API. The relevant details can be found in the documentation at https://developer-docs.amazon.com/sp-api/docs/reports-api-v2021-06-30-reference. Specifically, you would want to use the `GET_SALES_AND_TRAFFIC_REPORT` report. [see the page](https://developer-docs.amazon.com/sp-api/docs/report-type-values-analytics#seller-retail-analytics-reports)

It's worth noting that gaining access to the Seller API might not be a straightforward process. Refer to the documentation at  https://developer-docs.amazon.com/sp-api/docs/sp-api-registration-overview for information on accessing Seller Retail Analytics reports.

If you encounter any challenges or have specific questions about using the Seller Partner API, feel free to provide more details, and I'll do my best to assist you.
